src/main/java/org/symphonyoss/integration/webhook/salesforce/SalesforceWebHookIntegration.java

```diff
@@ -66,6 +72,12 @@ public void onConfigChange(IntegrationSettings settings) {
 
   @Override
   public String parse(WebHookPayload input) throws WebHookParseException {
+    String messageML;
```

Comment:
Why did you move this declaration?  In general, it is best to keep scope of variables as narrow as possible.  Previously, it was declared when at the time it is first used.  Putting variable declaration at the top of the method is very 'C'/'C++' like.